config/organise_runfolder.yml

```diff
@@ -0,0 +1,43 @@
+---
+#Input: runfolder_name
+
+#Currently named runfolderpath and runfolder to harmonize with rnaseq config by Monika, but for specific runfolder delivery, technically, these could be "merged" to one.
+runfolderpath: /proj/ngi2016001/incoming
+runfolder: <RUNFOLDERPATH>/<RUNFOLDER_NAME>
+organised: <RUNFOLDER>/Projects/
+
+#current path is /proj/ngi2016001/incoming/<RUNFOLDER>/Projects/<PROJECT>/<RUNFOLDER>/
+#The following is based on that we are to keep the same directory structure as above.
+files_to_organize:
+
+        #The fastq files
+        - source: <RUNFOLDER>/Unaligned/*  
+          destination: <ORGANISED>/(?P=projectid)/<RUNFOLDER_NAME>/Sample_(?P=samplename)/
```

Comment:
Not sure it works to mix both the `<VARIABLE>` and `(?P=variable)` syntax here. Maybe we should just use the [Python format syntax](https://peps.python.org/pep-3101/) when doing substitution instead of regex matching. We should discuss this and how to declare variables in general during the next meeting.